tests/lib/state_test.py

```diff
@@ -723,6 +723,30 @@ def test_merge_config_dns_empty_current():
     assert desire_state.config_dns == dns_config[DNS.CONFIG]
 
 
+def test_state_verify_dns_same():
+    dns_config = _get_test_dns_config()
+    desire_state = state.State({DNS.KEY: dns_config})
+    current_state = state.State({DNS.KEY: dns_config})
+
+    desire_state.verify_dns(current_state)
+
+
+def test_state_verify_dns_revert():
```

Comment:
Did you mean `reversed order` instead of `revert`?
Perhaps call it `test_verify_dns_same_entries_different_order`